src/agent.ts

```diff
@@ -35,7 +35,7 @@ export class Agent {
   private _wraps: WrapLibrary.Wrap[];
   private _library: WrapLibrary.Reader;
   private _client: PolywrapClient;
-
+  private _knownWraps: { [key: string]: {description: string, repo: string} };
```

Comment:
I think it probably makes more sense to hang-onto these wraps within the wrap library, instead of here. Also it's better to use an interface type instead of creating a type inline (i.e. `{ description: string, repo: string }`)